src/utils/components/DiskModal/BlankDiskModal.tsx

```diff
@@ -32,6 +34,9 @@ const BlankDiskModal: FC<V1DiskModalProps> = ({
   const isVMRunning = isRunning(vm);
 
   const isEditDisk = !isEmpty(editDiskName);
+  const isCreated = !isEmpty(createdPVCName);
+  const namespace = getNamespace(vm);
+  const [pvc] = usePVCDiskSource(createdPVCName, namespace);
```

Comment:
why calling this in every modal and not just send it as prop? 1 call and not every modal change, which is option change another call..